src/Loader.php

```diff
@@ -112,7 +112,7 @@ protected function renderView(): void
     public function jsLoad($args = [], $apiConfig = [], $storeName = null)
     {
         return $this->js()->atkReloadView([
-            'uri' => $this->cb->getJsUrl(),
+            'uri' => $this->cb->getUrl(),
```

Comment:
not loaded with JS?

Reply:
Terminate with js but uses Callback and not JsCallback.

Reply:
you are right